docs-gen/content/rule_set/instances.md

```diff
@@ -10,17 +10,39 @@ quite often there is a need to repeat branches and data entries
 branches and data entries in the specification an instance-concept is supported.
 Instances remove the need of repeating definitions, by defining at the node itself how often it occurs in
 the resulting tree. They are meant as a short-cut in the specification and
-interpreted by the tools. As an example is shown below for doors:
+interpreted by the tools.
+
+The instance names defined in VSS can be seen as labels
+to reference particular instances. There is not necessarily a 1:1 relation between
+VSS instance identifiers and physical instances. A typical example could be instance identifiers
+for doors, windows and seats. For a left-hand drive vehicle, the driver seat may be referenced both
+as `DriverSide` and as `Left`. VSS does not provide any syntax to specify that VSS instance `A`
+(for this particular vehicle) actually refers to the same physical instances as `C`.
+It is up to the VSS implementation and deployment to manage this relationship, for example
+by a configuration file.
+
+
+As an example is shown below for doors. In VSS four instances of doors are specified for each row,
```

Comment:
Review phrasing. Not clear what is meant by "As an example is shown below."

"In VSS" -> "In VSS,"